apps/console/src/features/core/components/header.tsx

```diff
@@ -170,6 +171,16 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
             } );
     }, [ tenantDomain, associatedTenants ]);
 
+    useEffect(() => {
```

Comment:
Better to place useEffects with empty dependancy array, prior to useEffects with dependancies.
And let's add a doc comment for the useEffect mentioning the purpose.

Reply:
> Better to place useEffects with empty dependancy array, prior to useEffects with dependancies. 

just asking to clarify. Do we have such a standard? 

> And let's add a doc comment for the useEffect mentioning the purpose.

Addressed with 8ebcfb1